crashtracker/src/crash_info.rs

```diff
@@ -243,10 +243,12 @@ impl CrashInfo {
     /// SIGNAL SAFETY:
     ///     I believe but have not verified this is signal safe.
     pub fn to_file(&self, path: &Path) -> anyhow::Result<()> {
-        let file =
-            File::create(path).with_context(|| format!("Failed to create {}", path.display()))?;
+        let binding = path.as_os_str().to_string_lossy();
+        let path = binding.strip_prefix("file://").unwrap_or(&binding);
```

Comment:
Removed, it was not needed.
I think I did that because I copied `ddog_Endpoint_file(to_slice_c_char("file://tmp/test"))` from `examples/ffi/crashinfo.cpp`. 
But I think the correct code is `ddog_Endpoint_file(to_slice_c_char("/tmp/test"))`.

I updated the example.